get-started/index.md

```diff
@@ -248,38 +246,38 @@ auth implementation.
 applications, feel free to [continue to the next step](#open-the-cryostat-web-ui).
 
 ```bash
 $ oc new-app --docker-image=quay.io/andrewazores/quarkus-test:0.0.10
 $ oc patch svc/quarkus-test -p '{"spec":{"$setElementOrder/ports":[{"port":9097},{"port":8080}],"ports":[{"name":"jfr-jmx","port":9097}]}}'
 ```
 
-This is a Quarkus container in JVM mode with JMX enabled and pre-configured to
-listen on port 9097.  After deploying the container we patch its service to
-name the 9097 service port `jfr-jmx`. Cryostat will detect and use this port
+This is a **Quarkus** container in JVM mode with JMX enabled and pre-configured to
+listen on `port 9097`.  After deploying the container we patch its service to
+name the `9097` service port `jfr-jmx`. **Cryostat** will detect and use this port
 to determine that this is a compatible Java application that it should monitor.
 
 #### [Configuring Applications](#configuring-applications)
-There are three methods of configuring your Java applications so that Cryostat is able to discover and monitor them:
+There are **three** methods of configuring your Java applications so that **Cryostat** is able to discover and monitor them:
 
 1. [using the Cryostat Agent for discovery and connectivity](#using-the-cryostat-agent)
 2. [using platform mechanisms for discovery and Java Management Extensions (JMX) for connectivity](#using-jmx)
 3. [using the Cryostat Agent for discovery and JMX for connectivity](#using-the-cryostat-agent-with-jmx)
 
 The following sections will briefly explain how to accomplish each of these approaches by example. For simplicity the examples will assume your application
-is built with Maven, packaged into an image with a `Dockerfile`, and running in Kubernetes, but the instructions will be similar for other toolchains and platforms as well.
+is built with **Maven**, packaged into an image with a `Dockerfile`, and running in **Kubernetes**, but the instructions will be similar for other toolchains and platforms as well.
 
 ##### [Using the Cryostat Agent](#using-the-cryostat-agent)
 
 [The Cryostat Agent](/guides/#using-the-cryostat-agent)
-is compatible with Cryostat versions 2.3.0 and newer, and application JDKs 11 and newer. If you are using an older version of Cryostat, we recommend upgrading to ensure compatibility.
-If your application uses a recent version of JDK8 with JFR support, please either upgrade to JDK11+ or [continue to the next section](#using-jmx)
-to learn how to configure your application without the Cryostat Agent.
+is compatible with **Cryostat** versions 2.3.0 and newer, and application **JDKs 11** and newer. If you are using an older version of **Cryostat**, we recommend upgrading to ensure compatibility.
+If your application uses a recent version of **JDK8** with JFR support, please either upgrade to **JDK11+** or [continue to the next section](#using-jmx)
+to learn how to configure your application without the **Cryostat** Agent.
```

Comment:
If elsewhere **Cryostat web** gets styled as a two-word term, I think **Cryostat Agent** deserves that as well.